deploy/kubernetes/dolphinscheduler/values.yaml

```diff
@@ -506,10 +506,15 @@ master:
     MASTER_TASK_COMMIT_INTERVAL: "1s"
     # -- master state wheel interval, the unit is second
     MASTER_STATE_WHEEL_INTERVAL: "5s"
-    # -- Master max cpuload avg, only higher than the system cpu load average, master server can schedule
-    MASTER_MAX_CPU_LOAD_AVG: "1"
-    # -- Master reserved memory, only lower than system available memory, master server can schedule, the unit is G
-    MASTER_RESERVED_MEMORY: "0.3"
+    MASTER_SERVER_LOAD_PROTECTION_ENABLED: true
```

Comment:
Lack of comments

Reply:
If deployed through k8s, this default value should be able to be set to false.

Reply:
Done.